content/resources/ebook/cnj.md

```diff
@@ -0,0 +1,481 @@
+---
+title: "Fulfilling the Vision for Open Source, Cloud Native Java"
+seo_title: "Fulfilling the Vision for Open Source, Cloud Native Java | Jakarta EE"
+headline: "Fulfilling the Vision for Open Source, Cloud Native Java"
+tagline: "Leveraging the Strength of a United Community to Advance Enterprise Java"
+date: 2019-02-27T16:10:38-04:00
```

Comment:
Everything looks good to me! Just about this date, do we need to use a more recent date here?

Reply:
I don't think it matters that much since the date is in the past but I can change it anyway